compute_endpoint/globus_compute_endpoint/exception_handling.py

```diff
@@ -62,3 +70,24 @@ def get_result_error_details(exc: BaseException | None = None) -> tuple[str, str
         "RemoteExecutionError",
         "An error occurred during the execution of this task",
     )
+
+
+def handle_auth_errors(f: t.Callable) -> t.Callable:
+    @functools.wraps(f)
+    def wrapper(*args, **kwargs):
+        try:
+            return f(*args, **kwargs)
+        except globus_sdk.AuthAPIError as e:
+            msg = textwrap.dedent(
+                f"""
+                Compute CLI Error: An Auth API error occurred.
+                HTTP status:       {e.http_status}
+                code:              {e.code}
+                message:           {e.text}
+                """
+            )
+            log.warning(msg)
+            click.secho(msg)
+            sys.exit(os.EX_NOPERM)
```

Comment:
Heh; seems almost a shame to use `secho` and _not_ have any colors.  ;-)

Reply:
Hah, good catch.